API.md

```diff
@@ -1,7 +1,7 @@
 
 ## Introduction
 
-**yar** add session support to hapi - a persistent state across multiple browser requests using an [iron](https://github.com/hapijs/iron) encrypted cookie and server-side storage. **yar** tries to fit session data into a session cookie based  on a configured maximum size. If the content is too big to fit, it uses server storage via the [hapi plugin cache](http://hapi.dev/api#servercacheoptions) interface.
+**yar** add session support to hapi - a persistent state across multiple browser requests using an [iron](https://github.com/hapijs/iron) encrypted cookie and server-side storage. **yar** tries to fit session data into a session cookie based  on a configured maximum size. If the content is too big to fit, it uses server storage via the [hapi plugin cache](https://hapi.dev/api#-servercacheoptions) interface.
```

Comment:
Little-known fact: the API docs have `<a name="" />` tags manually placed on headings so that the anchors have prettier, and stable links:
```suggestion
**yar** add session support to hapi - a persistent state across multiple browser requests using an [iron](https://github.com/hapijs/iron) encrypted cookie and server-side storage. **yar** tries to fit session data into a session cookie based  on a configured maximum size. If the content is too big to fit, it uses server storage via the [hapi plugin cache](https://hapi.dev/api#server.cache()) interface.
```

Reply:
Good to know. I always wondered if we had stable links!